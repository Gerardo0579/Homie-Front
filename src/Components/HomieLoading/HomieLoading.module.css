.loadingWrapper {
    margin-top: 10em;
    width: 100%;
    height: 100%;
    display: flex;
    justify-content: center;
}

.spinner {
    margin-top: 30%;
    margin-bottom: 30%;
}

@media (max-width: 1030px) {
    .loadingWrapper {
        margin-top: 0;
    }
}