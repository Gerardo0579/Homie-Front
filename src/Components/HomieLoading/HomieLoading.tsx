import { Spin } from 'antd'
import { FC } from 'react'
import styles from './HomieLoading.module.css'

interface LoadingProps {
  tip: string
}
export const HomieLoading: FC<LoadingProps> = ({ tip }) => {
  return (
    <div className={styles.loadingWrapper}>
      <Spin className={styles.spinner} size="large" tip={tip} />
    </div>
  )
}
