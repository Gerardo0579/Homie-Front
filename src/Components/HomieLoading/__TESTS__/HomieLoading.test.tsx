import renderer from 'react-test-renderer'
import { HomieLoading } from '../HomieLoading'

const CONTENT: string = '1'

it('renders correctly', () => {
  const tree = renderer.create(<HomieLoading tip={CONTENT} />).toJSON()
  expect(tree).toMatchSnapshot()
})
