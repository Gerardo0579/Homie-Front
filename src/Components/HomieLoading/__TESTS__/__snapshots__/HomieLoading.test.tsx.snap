// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loadingWrapper"
>
  <div
    aria-busy={true}
    aria-live="polite"
    className="ant-spin ant-spin-lg ant-spin-spinning ant-spin-show-text spinner"
  >
    <span
      className="ant-spin-dot ant-spin-dot-spin"
    >
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
      <i
        className="ant-spin-dot-item"
      />
    </span>
    <div
      className="ant-spin-text"
    >
      1
    </div>
  </div>
</div>
`;
